### PR TITLE
Change summary panel

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Update the styling and wording on the panels that are shown on summaries.

### How to review 
Check that all summaries in the household schema have blue panels with the correct wording from the Trello card

### CODE REVIEW

**Review by iwootten — APPROVED** [review]
👍

**Review by LJBabbage — APPROVED** [review]
All section summaries changed to the grey version when run

File: templates/summary.html [hunk 1]
```diff
@@ -21,11 +21,10 @@ <h1>{{ _("Check your answers and submit") }}</h1>
 
     <h1>{{content.summary.title}}</h1>
       {% call onsPanel({
-        "type": "success",
```
Comment (iwootten) [inline templates/summary.html:24]: You shouldn't need to define the type as the default is information?

Comment (rmccar) [reply templates/summary.html:24]: Cool, sorted that. I tested it and it works for me

**Review by iwootten — CHANGES_REQUESTED** [review]
Blocking this until someone senior makes the call whether we're allowed to fix this issue.

**Review by iwootten — APPROVED** [review]
Approving this based on the panel changes.

**Pull request merged**